asdf_astropy/converters/unit/tests/test_quantity.py

```diff
@@ -7,6 +9,13 @@
 from numpy.testing import assert_array_equal
 
 
+def asdf_open_memory_mapping_kwarg(memmap: bool) -> dict:
```

Comment:
This package can use the (sometimes confusing) `minversion` from astropy:
https://github.com/astropy/asdf-astropy/blob/dab5b4d73ce3f75e60e5b29e75eb3e811dfd8319/asdf_astropy/converters/transform/core.py#L134